configmap/parse.go

```diff
@@ -141,7 +141,11 @@ func AsDuration(key string, target *time.Duration) ParseFunc {
 func AsStringSet(key string, target *sets.String) ParseFunc {
 	return func(data map[string]string) error {
 		if raw, ok := data[key]; ok {
-			*target = sets.NewString(strings.Split(raw, ",")...)
+			urlArr := strings.Split(raw, ",")
```

Comment:
nit: `urlArr` isn't quiet accurate here. The variable doesn't contain URLs per se. Personally, I would've chosen `splitted` in this case.